starter_code/person.js

```diff
@@ -1,5 +1,12 @@
+"use strict";
+const Elevator = require('./elevator.js');
 class Person {
-  constructor(name, originFloor, destinationFloor){
+
```

Comment:
Here is how I should create the class:

class Person {
  constructor(name, originFloor, destinationFloor) {
    this.name = name;
    this.originFloor = originFloor;
    this.destinationFloor = destinationFloor;
  }
}

module.exports = Person;